productsapp/App.js

```diff
@@ -1,31 +1,38 @@
 import * as React from 'react';
 import { Text, View, StyleSheet } from 'react-native';
 import { Constants } from 'expo';
+import { createStackNavigator, createAppContainer } from 'react-navigation';
 
-// You can import from local files
 import Login from './components/Login';
 import ProductsList from './components/ProductsList';
+import Product from './components/Product';
 
-// or any pure javascript modules available in npm
 import { Card } from 'react-native-paper';
 
-export default class App extends React.Component<{}> {
+class App extends React.Component<{}> {
   render() {
     return (
-      <View style={styles.container}>
-          <Login />
-      </View>    
+      <View>
+        <AppStackNavigator />
+      </View>
     );
   }
 }
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    paddingTop: Constants.statusBarHeight,
-    backgroundColor: '#ecf0f1',
-    padding: 8,
-
+const AppStackNavigator = createStackNavigator({
+  LoginScreen: Login,
+  ProductsListScreen: {
+    screen: ProductsList,
+    navigationOptions: ({ navigation }) => ({
```

Comment:
if you don't use `navigation ` object inside navigationOptions, you can write like this

ProductsListScreen: {
    screen: ProductsList,
    navigationOptions: {
      title: `ProductsList`,
    },
  },